mycroft/skills/settings.py

```diff
@@ -90,21 +111,26 @@ def get_local_settings(skill_dir, skill_name) -> dict:
     return skill_settings
 
 
-def save_settings(skill_dir, skill_settings):
+def save_settings(skill_dir, skill_name, skill_settings):
```

Comment:
Let's not change the logic and interface here, it should do the save in the specified directory. The correct directory should be sent in through the skill_directory argument.